doc/code_style/mom.h

```diff
@@ -18,13 +18,13 @@
 
 // Includes are sorted alphabetically,
 // global includes first, then system inclusions:
-#include <cstddef>
-#include <cstdint>
-
```

Comment:
Is there a switch for this in clang-format? Didn't find one. Should we consider changing this in the template?